<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetCategories">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="categoriesLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Категории</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollAreaProducts">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>641</width>
      <height>500</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>639</width>
       <height>498</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="adminWindowNameLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Админ панель</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="productsInCatelabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Товары в категории </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateCategory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать категорию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteCategory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить категорию</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateProduct">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>590</y>
      <width>311</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать товар</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>948</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
